pkg/process/metadata/parser/dockerproxy.go

```diff
@@ -15,8 +15,9 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/util/log"
 )
 
-// Filter keeps track of every docker-proxy instance and filters network traffic going through them
-type Filter struct {
+// DockerProxy keeps track of every docker-proxy instance and filters network traffic going through them
+// TODO: Add a mechanism to clean up the maps
```

Comment:
I think one thing I'm struggling with on the Agent side is how we do caching. I think these maps are great candidates for TTL and/or LRU based caches to avoid unbounded memory growth.